go/vt/sqlparser/ast.go

```diff
@@ -2780,9 +2790,6 @@ func (node *Show) Format(buf *TrackedBuffer) {
 		}
 	}
 
-	if node.HasOnTable() {
-		buf.Myprintf(" on %v", node.OnTable)
```

Comment:
I couldn't find any valid use of `on <tbl_name>` in SHOW statements in MySQL's reference and verified all tests pass without this. 
https://dev.mysql.com/doc/refman/5.7/en/show.html

Reply:
Huh, weird, wonder what this was supposed to be for